Specs/DateTime/English/DateTimeModelComplexCalendar.json

```diff
@@ -6404,5 +6404,60 @@
         }
       }
     ]
+  },
+  {
+    "Input": "The date should be 05-Aug-2016",
+    "Context": {
+      "ReferenceDateTime": "2018-10-24T12:00:00"
+    },
+    "NotSupportedByDesign": "java",
+    "Results": [
+      {
+        "Text": "05-aug-2016",
+        "Start": 19,
+        "End": 29,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2016-08-05",
+              "type": "date",
+              "value": "2016-08-05"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Are you available on Monday morning from 10am to 12pm",
```

Comment:
Does "Monday afternoon from 4 to 6" work?

Reply:
This specific case is not supported in this PR as the extraction part also needs to refine to support this case. This support would come in the upcoming PR.
But this PR enable the differentiate between "Monday 4-6" and "Monday afternoon 4-6".